styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #F6F6F6;
+}
+
+h1 {
+  text-align: center;
+  text-transform: uppercase;
+  text-shadow: 3px 0 yellow;
+  font-family: helvetica;
+}
+
+nav {
+  text-align: center;
+  font-family: verdana;
+}
+
+nav a {
+  text-decoration: none;
+  text-shadow: 2px 0 yellow;
+  color: black;
+}
+
+a:hover {
+  color: red;
+}
+
+a.museum {
```

Comment:
This was kind of a last resort. I'm actually running into similar issues in the current homework assignment as well. Logically, when looking at it, there's no reason that the more broad classes should not be working in my external style sheet. But for some reason, the changes I was/am making don't always apply. The more specific tags/classes structure was just a workaround I arrived at. 

Long story short, I definitely had the same thought you did. I'll take a look at those articles. 

Thanks.